client/blocks/comment-detail/index.jsx

```diff
@@ -172,4 +188,32 @@ export class CommentDetail extends Component {
 	}
 }
 
-export default CommentDetail;
+const mapStateToProps = ( state, { commentId, siteId } ) => {
+	//const comment = getComment( state, siteId, commentId );
+	const comment = mockComment;
+
+	return {
+		authorAvatarUrl: comment.author.avatarUrl,
```

Comment:
In general we should guard against assuming that an object is well formed. For example let's say comment is `null` on an empty cache.

For more than one level, something like would do the trick.
```jsx
get( comment, 'author.avatarUrl' );
```

We may want to create new selectors if we end up using these properties in multiple places.